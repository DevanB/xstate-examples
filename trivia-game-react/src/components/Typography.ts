import tw from 'tailwind.macro'

export const H1 = tw.h1`antialiased font-sans text-2xl font-bold text-gray-800 text-center`

export const P = tw.p`antialiased font-sans text-base font-nomral text-gray-800 text-center`
