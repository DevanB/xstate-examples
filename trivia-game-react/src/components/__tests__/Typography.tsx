import {render} from '@testing-library/react'
import React from 'react'
import {H1, P} from '../Typography'

describe('<H1/>', () => {
  it('should match snapshot', () => {
    const {container} = render(<H1 />)
    expect(container).toMatchSnapshot()
  })
})

describe('<P/>', () => {
  it('should match snapshot', () => {
    const {container} = render(<P />)
    expect(container).toMatchSnapshot()
  })
})
