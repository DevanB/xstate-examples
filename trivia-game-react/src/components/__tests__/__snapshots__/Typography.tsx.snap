// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1/> should match snapshot 1`] = `
<div>
  <h1
    class="Typography__H1-sc-1qcilcn-0 kUFxaM"
  />
</div>
`;

exports[`<P/> should match snapshot 1`] = `
<div>
  <p
    class="Typography__P-sc-1qcilcn-1 hAyRKx"
  />
</div>
`;
